exercises/concept/card-games/lists_test.py

```diff
@@ -61,9 +60,9 @@ def test_empty(self):
         want = []
 
         self.assertEqual(concatenate_rounds(rounds_1, rounds_2),
-            want,
-            msg=f'Expected {want} but got an incorrect result.'
-        )
+                         want,
+                         msg=f'Expected {want} but got an incorrect result.'
+                         )
```

Comment:
This indentation seems different than the indentation style [here](https://github.com/exercism/python/pull/2553/files#diff-1ac9d52a20cc974321f6dd16d3bcdc393030a6e253b062a7328df29fc7592928R88).

Reply:
This was an auto-format from PyCharm.  It lined the arguments to `self.asserEqual()` up on their own lines.  The guidance is actually fuzzy on this - either 4 indents or "lining up" will pass lint.  But we probably should be consistent within a file.